.prow/scripts/test-serving.sh

```diff
@@ -1,17 +1,17 @@
 #!/usr/bin/env bash
 
+MVN_OPTS='--batch-mode --also-make --projects serving'
+
 .prow/scripts/download-maven-cache.sh \
     --archive-uri gs://feast-templocation-kf-feast/.m2.2019-10-24.tar \
     --output-dir /root/
 
-# Skip Maven enforcer: https://stackoverflow.com/questions/50647223/maven-enforcer-issue-when-running-from-reactor-level
-mvn --projects serving --batch-mode --define skipTests=true \
-    --define enforcer.skip=true clean install
-mvn --projects serving --define enforcer.skip=true test
+mvn "$MVN_OPTS" --define skipTests=true clean install
+mvn "$MVN_OPTS" test
```

Comment:
The double quotes around `$MVN_OPTS` seem to break the test because the final command run includes extra single quotes:
```
mvn '--batch-mode --also-make --projects serving' --define skipTests=true clean install
```

I think we can remove install before test like what you suggested

Reply:
This seems to work fine
```
mvn $MVN_OPTS --define skipTests=true clean install
```

Reply:
Doh, I hope I never have a gun held to my head asked to get shell quoting right…

Updated to fix that and removed the `install` step. Thanks!